### PR TITLE
Error response fixes and tests

### PR DESCRIPTION
**Pull request by jingsting**
## Purpose
- Make sure useful rpc error responses are returned for unlockKeyFile
- Check and test all the modifier query with invalid ids
- Remove local-testnet since we no longer use it

## Approach
- Added error response when there is no corresponding keyfile found for unlockKeyFile
- Added test for querying modifiers with an id containing non base58 characters
- Removed local-testnet

## Closes
- #1532 
- #1538 
- #1530

### CODE REVIEW

File: common/src/main/scala/co/topl/attestation/keyManagement/KeyRing.scala [hunk 1]
```diff
@@ -215,18 +218,20 @@ class KeyRing[
...
-          require(listOfKeyfiles.size == 1, s"Cannot find a unique matching keyfile in $keyDirectory")
-          listOfKeyfiles.head
+    private def checkValid(address: Base58Data, password: Latin1Data): Try[KF] =
+      Try {
+        listKeyFiles()
+          .map {
+            _.filter {
+              _.address == address.decodeAddress.getOrThrow()
+            }
+          } match {
```
Comment (scasplte2) [inline common/src/main/scala/co/topl/attestation/keyManagement/KeyRing.scala:228]: I think you could do a `.toTry` here right?

**Review by scasplte2 — CHANGES_REQUESTED** [review]
looks good, lets go over with everyone and look at the .toTry question. Otherwise this would be good to go I think.

**Comment by scasplte2** [comment]
These are all the changes that you wanted to incorporate right @jingsting ?

**Comment by jingsting** [comment]
> These are all the changes that you wanted to incorporate right @jingsting ?

Yes, this has the changes that we wanted yesterday

**Comment by scasplte2** [comment]
👍 got it, going to merge

**Review by scasplte2 — APPROVED** [review]
🔥

**Pull request merged**